src/Atmos/Model/radiation.jl

```diff
@@ -24,7 +24,7 @@ end
 """
   StevensRadiation <: RadiationModel
```

Comment:
```suggestion
  DYCOMSRadiation <: RadiationModel
```
We should call this DYCOMS radiation to make clear where it is being used. Also, all of this should go to the DYCOMS driver, not `radiation.jl`. (This is not a proper radiative transfer model, just a very specific prescription used in only one case.)

Reply:
The only problem with moving this to the Dycoms driver is that this struct is used in the following three drivers: `ex_001_dycoms.jl`, `dycoms_sin_test.jl`, `ex_001_dycoms-imex.jl`, which means, we would have duplicate code in the drivers. Should I move and duplicate? Or keep in `radiation.jl`?

Reply:
there should be only 1 dycoms driver. The only one that is correct is contained in 

`DGmethods/compressible_Navier_Stokes/ex_001_dycoms-imex.jl`

in my reference branch that you are using for this merging process.